public/Get-DbaReplServer.ps1

```diff
@@ -60,9 +60,14 @@ function Get-DbaRepServer {
         if (Test-FunctionInterrupt) { return }
         foreach ($instance in $SqlInstance) {
             try {
-                # use System.Data instead of Microsoft.Data
-                $sqlconn = New-SqlConnection -SqlInstance $instance -SqlCredential $SqlCredential
-                New-Object Microsoft.SqlServer.Replication.ReplicationServer $sqlconn
+                $server = Connect-DbaInstance -SqlInstance $instance -SqlCredential $SqlCredential
```

Comment:
We need to be consistent here if we are not going to use `New-SqlConnection` anymore then it should be removed from ` Test-DbaReplLatency` as well, completely.

We only created that command to deal with RMO initially I recall. So if we don't require it anymore, get rid of it.

Reply:
I've opened and Issue for `Test-DbaReplLatency` that doesn't work at all for me and so I've added on that issue that it should be rewritten to not use `New-SqlConnection`. I'll try and get that picked up once this stuff is complete.

https://github.com/dataplat/dbatools/issues/9028